src/main/resources/lang/default.lang

```diff
@@ -1958,6 +1958,32 @@ environments:
 	the_end: end, the end
 	custom: custom
 
+
+entity potion causes:
```

Comment:
I'm worried about a lot of these types interfering with existing syntaxes
Have you tested to make sure these don't interfere in weird places?

Reply:
> I'm worried about a lot of these types interfering with existing syntaxes Have you tested to make sure these don't interfere in weird places?

Why would they interfere? the other syntaxes aren't using this type.

Reply:
Imagine `set {_s} to conduit`. Does it set it to the conduit item? The conduit entity potion cause?